src/db-driver.js

```diff
@@ -0,0 +1,45 @@
+const sqlLoader = require('@prairielearn/prairielib/sql-loader')
+const sqlDB = require('@prairielearn/prairielib/sql-db')
+const ERR = require('async-stacktrace')
+const config = require('./lib/config.js')
+const logger = require('./logger')
+const fs = require('fs')
+
+var dbDriver = module.exports
+
+const pgConf = {
+  host: config.postgresqlHost,
+  port: config.postgresqlPort,
+  user: config.postgresqlUser,
+  password: config.postgresqlPassword,
+  database: config.postgresqlDatabase,
+}
+
+dbDriver.initDB = function(callback, idleErrCb) {
```

Comment:
It looks like `callback` is never invoked if there isn't an error?

Reply:
Sorry, on my phone, line 34.